samples/AlarmBot/Responses/AddAlarmResponses.cs

```diff
@@ -33,44 +31,44 @@ public static void ReplyWithConfused(IBotContext context)
 
         public static void ReplyWithCancelPrompt(IBotContext context, Alarm alarm)
         {
-            context.Batch().Reply(ResponseHelpers.ReplyWithSuggestions(context, "Cancel Alarm?", $"Did you want to cancel the alarm?\n\n{AlarmDescription(context, alarm)}", YesNo));
+            context.Batch().Reply(context.ReplyWithSuggestions("Cancel Alarm?", $"Did you want to cancel the alarm?\n\n{AlarmDescription(context, alarm)}", YesNo));
```

Comment:
This code reads very strangly. As I read it, I assume we're actualy replying twice. Once as part of a Batch() operation, and once as part of an immediate / synchronous operation. 

We're planning to pull all the Batch() stuff out (probably tomorrow), so I could clean this up then.

Reply:
Sure, this was in the code previously